src/utils/utils-consts.js

```diff
@@ -1,5 +1,6 @@
+
 const VENDOR_PREFIXES = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
-const TEST_ELEMENT = document.createElement('div');
+const TEST_ELEMENT = typeof document === "undefined" ? {style: {}} : document.createElement('div') 
```

Comment:
@younkue semicolon is omitted.
and why style is defined where document type is undefiend.

Reply:
use style of TEST_ELEMENT in get-touchaction-props.js